libraries/Wire/Wire.cpp

```diff
@@ -226,7 +226,7 @@ int TwoWire::available(void)
     {
         // yielding here will not make more data "available",
         // but it will prevent the system from going into WDT reset
-        optimistic_yield(1000);
+        optimistic_yield(10000UL);
```

Comment:
:) Here's some old cut-n-paste, I think.  Same comment as UDP and TCP.  Stick w/100.